pkg/services/reputation/eigentrust/storage/daughters/calls.go

```diff
@@ -52,13 +53,44 @@ func (x *Storage) DaughterTrusts(epoch uint64, daughter reputation.PeerID) (*Dau
 	return s.daughterTrusts(daughter)
 }
 
+// AllDaughterTrusts returns daughter iterator for the epoch.
+//
+// Returns false if there is no data for the epoch and daughter.
+func (x *Storage) AllDaughterTrusts(epoch uint64) (*daughterStorage, bool) {
```

Comment:
Exported method returns value of unexported type?

Reply:
outdated, was fixed yesterday